GameData/BeyondHome/BeyondHome_InstallChecker-SOURCE.cs

```diff
@@ -1,191 +1,180 @@
+            DoLogThingsForMe("BICH stands for Beyondhome Installation Checker. If you're reading this, you're evaluating what went wrong or stumbled across it when something else went wrong.");
+            DoLogThingsForMe("Thank you for using Beyond Home. Below you can find out what went wrong (or right!)");
+            string path = Application.dataPath.Remove(Application.dataPath.Length - 12, 12) + "GameData/";
+            DoLogThingsForMe("GameData directory is " + path);  //Well, I sure hope so
+            Debug.Log("");
+            DoLogThingsForMe("Displaying errors, warnings and notices below:");
+            if (memorysize > 4.9)
+            {
+                errors = errors + ("<color=#00F200>Recommended RAM: 5GB  -  Installed RAM: " + memorysize.ToString() + "\n");
+                DoLogThingsForMe("  - RAM meets or exceeds requirement");
+            }
+            else
+            {
+                errors = errors + ("<color=#EF0013>Recommended RAM: 5GB  -  Installed RAM: " + memorysize.ToString() + "\n");
+                DoLogThingsForMe("  - RAM does not meet the requirement");
+            }
+            if (gpumemorysize > 1)
+            {
+                errors = errors + ("<color=#00F200>Recommended VRAM: 1GB  -  Installed VRAM: " + gpumemorysize.ToString() + "\n");
+                DoLogThingsForMe("  - VRAM meets or exceeds the requirement");
+            }
+            else
+            {
+                errors = errors + ("<color=#00F200>Recommended VRAM: 1GB  -  Installed VRAM: " + gpumemorysize.ToString() + "\n");
+                DoLogThingsForMe("  - VRAM does not meet the requirement");
+            }
+
+            if (System.IO.File.Exists(path + "BeyondHome/BeyondHome_LICENSE.txt") == false)
+            {
+                errors = errors + "<color=#FF3F00>ERROR: Beyond Home is not installed!</color>\n";   //Nobody goes into the mod folder and removes the license file, so this indicates a bad install
+                DoLogThingsForMe("  - Beyond Home is not even installed...?");
+                errorcount++;
+            }
+            if (System.IO.File.Exists(path + "Kopernicus/Plugins/Kopernicus.dll") == false)
+            {
+                errors = errors + "<color=#FF3F00>ERROR: Kopernicus is not installed!</color>\n";
+                DoLogThingsForMe("  - Kopernicus is not installed");
+                errorcount++;
+            }
+            if (System.IO.File.Exists(path + "ModularFlightIntegrator/ModularFlightIntegrator.dll") == false)
+            {
+                errors = errors + "<color=#FF3F00>ERROR: Modular Flight Integrator is not installed!</color>\n";
+                DoLogThingsForMe("  - MFI is not installed");
+                errorcount++;
+            }
+//            if (System.IO.File.Exists(path + "Sigma/Replacements/SkyBox/Plugins/SigmaReplacementsSkyBox.dll") == true)

```

Comment:
This section here is broken, not sure why so i commented it out until i find the issue